landing_page/app/javascript/components/HelloWorld.tsx

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react'
+
+type MyProps = {
```

Comment:
Can you think of a better name than 'MyProps'? What if every component called their prop types this? A useful naming convention for props that I have seen is to use "<classOrFuction name>Props". Then at least the props name will be unique and easily identifiable as being related to the functional or class component.

Reply:
Resolved in commit aca6023